UI/menu/expanding.html

```diff
@@ -1,12 +1,17 @@
 <?lsmb# HTML Snippet, for import only ?>
         <div style="text-align: center"><a target="_blank" href="http://ledgersmb.org/">
-            <img class="cornderlogo" src="images/ledgersmb_small.png" width="100" height="50" border="1" alt="LedgerSMB" />
+            <img class="cornerlogo" src="images/ledgersmb_small-trans.png"
         </a></div>
         <div id="version_info"><?lsmb text("LedgerSMB [_1]", version) ?></div>
-        <div id="company_info"><?lsmb text("Logged into [_1]", company) ?></div>
-        <div id="login_info"><?lsmb text("Logged in as [_1]", login) ?></div>
-        <ul id="top_menu" class="menu_open">
-        <?lsmb old_level = 0 ?>
```

Comment:
I think the tests fail because you probably didn't mean to remove the last two rows. (The test searches for a page element which is called "top_menu".)